data/formatters/generic.yaml

```diff
@@ -14,6 +14,34 @@ short_message:
 - '{http_request}'
 - 'from: {ip_address}'
 ---
+type: 'conditional'
+data_type: 'aws:elb:access'
+message:
+- 'type: {type}'
+- 'elb: {elb}'
```

Comment:
this abbreviation will be lost in the volume of data, maybe "Elastic Load Balancer resource identifier" or "ELB resource identifier" ?

Reply:
Done